<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="graphql.execution.DataFetcherExceptionHandlerTest" tests="1" skipped="0" failures="0" errors="0" timestamp="2019-10-01T16:24:16" hostname="pc1295" time="0.015">
  <properties/>
  <testcase name="integration test to prove custom error handle can be made" classname="graphql.execution.DataFetcherExceptionHandlerTest" time="0.015"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
